R/vol-surface-class.R

```diff
@@ -0,0 +1,160 @@
+#' VolSurface class
+#'
+#' The `VolSurface`` class is designed to represent volatility surfaces.
+#' Checks whether the dimentions of the surface correspond to the right type of axis: moneyness, delta or strike
+#'
+#' @param reference_date `Date` value that represents the as of date
+#' @param surface_type type of the vol dimentions. Available types: "strike/term"
```

Comment:
spelling mistake here.

Reply:
fixed dimensions spelling in line 4 and 7.